### PR TITLE
document the ExtraHosts parameter for /containers/create for the remote ...

### PR DESCRIPTION
**Pull request by mcfiredrill**
...API

I think this was added from version 1.15, so I added it to version 1.15, 1.16, and 1.17.

### CODE REVIEW

**Comment by jfrazelle** [comment]
Can you please sign your commits following these rules:

https://github.com/docker/docker/blob/master/CONTRIBUTING.md#sign-your-work

The easiest way to do this is to amend the last commit:

`git commit --amend -s --no-edit && git push -f`

**Comment by mcfiredrill** [comment]
@jfrazelle done, thanks.

**Comment by SvenDowideit** [comment]
LGTM - @fredlf @jamtur01

File: docs/sources/reference/api/docker_remote_api_v1.15.md [hunk 1]
```diff
@@ -148,6 +148,7 @@ Create a container
                "Privileged": false,
                "Dns": ["8.8.8.8"],
                "DnsSearch": [""],
+	       "ExtraHosts": null,
```
Comment (jamtur01) [inline docs/sources/reference/api/docker_remote_api_v1.15.md]: Spacing?

File: docs/sources/reference/api/docker_remote_api_v1.15.md [hunk 2]
```diff
@@ -220,6 +221,8 @@ Json Parameters:
         a boolean value.
   -   **Dns** - A list of dns servers for the container to use.
   -   **DnsSearch** - A list of DNS search domains
+  -   **ExtraHosts** - A list of hostnames/ip mappings to be added to the
```
Comment (jamtur01) [inline docs/sources/reference/api/docker_remote_api_v1.15.md]: IP

File: docs/sources/reference/api/docker_remote_api_v1.15.md [hunk 3]
```diff
@@ -220,6 +221,8 @@ Json Parameters:
...
+      container's /etc/host file. Specified in the form `["hostname:ip"]`.
```
Comment (jamtur01) [inline docs/sources/reference/api/docker_remote_api_v1.15.md]: \`/etc/host\`

File: docs/sources/reference/api/docker_remote_api_v1.16.md [hunk 4]
```diff
@@ -148,6 +148,7 @@ Create a container
                "Privileged": false,
                "Dns": ["8.8.8.8"],
                "DnsSearch": [""],
+	       "ExtraHosts": null,
```
Comment (jamtur01) [inline docs/sources/reference/api/docker_remote_api_v1.16.md]: Spacing

File: docs/sources/reference/api/docker_remote_api_v1.16.md [hunk 5]
```diff
@@ -220,6 +221,8 @@ Json Parameters:
         a boolean value.
   -   **Dns** - A list of dns servers for the container to use.
   -   **DnsSearch** - A list of DNS search domains
+  -   **ExtraHosts** - A list of hostnames/ip mappings to be added to the
```
Comment (jamtur01) [inline docs/sources/reference/api/docker_remote_api_v1.16.md]: IP

File: docs/sources/reference/api/docker_remote_api_v1.16.md [hunk 6]
```diff
@@ -220,6 +221,8 @@ Json Parameters:
...
+      container's /etc/host file. Specified in the form `["hostname:ip"]`.
```
Comment (jamtur01) [inline docs/sources/reference/api/docker_remote_api_v1.16.md]: \`/etc/host\`

File: docs/sources/reference/api/docker_remote_api_v1.17.md [hunk 7]
```diff
@@ -149,6 +149,7 @@ Create a container
                "ReadonlyRootfs": false,
                "Dns": ["8.8.8.8"],
                "DnsSearch": [""],
+	       "ExtraHosts": null,
```
Comment (jamtur01) [inline docs/sources/reference/api/docker_remote_api_v1.17.md]: As above etc

**Comment by jamtur01** [comment]
Needs some cleanup before docs are okay.

**Comment by mcfiredrill** [comment]
@jamtur01 I've made the changes you requested. Shall I squash my commits?

**Comment by jamtur01** [comment]
@mcfiredrill Please do.

**Comment by mcfiredrill** [comment]
@jamtur01 done!

**Comment by jamtur01** [comment]
LGTM

File: docs/sources/reference/api/docker_remote_api_v1.15.md [hunk 8]
```diff
@@ -220,6 +221,8 @@ Json Parameters:
         a boolean value.
   -   **Dns** - A list of dns servers for the container to use.
   -   **DnsSearch** - A list of DNS search domains
+  -   **ExtraHosts** - A list of hostnames/IP mappings to be added to the
+      container's `/etc/host` file. Specified in the form `["hostname:IP"]`.
```
Comment (tianon) [inline docs/sources/reference/api/docker_remote_api_v1.15.md]: This should be /etc/host**s**...

**Pull request merged**